src/services/cache-service.ts

```diff
@@ -32,9 +32,12 @@ export class CacheService {
         this._cacheContainer[key] = [
             validUntil,
             loadingFunction().then(response => {
-                const cachedResponse = Promise.resolve(response);
-                this._cacheContainer[key] = [validUntil, cachedResponse];
-                return cachedResponse;
+                const cachedResponse = Promise.resolve(response)
```

Comment:
Is there a reason why you don't add semicolons at the end of the lines?

Reply:
Well. They are not necessary. Neither in JavaScript nor in TypeScript.

... I read a few years ago that semicolons improve the results of minifying, but at that time - if it ever was - it was already no longer true: Minifiers happily insert their own whitespaces and semicolons as they please, nowadays.

There is one specific case where a semicolon is _really_ necessary, if memory serves: declaring an [Immediately Invoked Function Expression](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).
```javascript
const someDefinition = "bla"
;(function(){ // arcane parsing error without semicolon
  <function body>
})()
```
But in this case, I think I'd rather write the separator (semicolon) in front, as would be customary, say, in SQL: It is required by the second expression, so into the line of the second expression it goes!

Given that the vast majority of code does not need or use semicolons for anything - why would I write them.

(FWIW: I only learned about IIFEs this year during a deciated JavaScript course, after having continuously used TypeScript for the last 4.5 years - they seem to be pretty much superceded by lambdas in TypeScript and indeed in ECMAScript, making this use case even more arcane and possibly deprecated.)